database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -14,13 +14,13 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
+            $table->integer('user_id');
             $table->id();
-            $table->string('name');
+            $table->boolean('is_admin');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->string('name');
             $table->string('password');
-            $table->rememberToken();
-            $table->timestamps();
```

Comment:
ayaw lang ni iremove, importante na naa tay created_at and update_at column para ma track nato ang pag save sa record

Reply:
Commited the requested changes. Please recheck.